cli/js/web/headers.ts

```diff
@@ -97,12 +96,22 @@ class HeadersBase {
     return `Headers {${output}}`;
   }
 
+  cookies(): IterableIterator<string> {
+    return this[cookieMap].values();
+  }
```

Comment:
This method isn't in the web API
https://developer.mozilla.org/en-US/docs/Web/API/Headers

Reply:
I know, I mentioned that but it was hard to implement it without extending it a bit. Although a better alternative could be to use a `symbol`, maybe in the Deno namespace.

```
[Deno.cookiesIterator](): IterableIterator<string> {
    return this[cookieMap].values();
}
```
That way is web compatible & can be iterated in `std/http/io.ts`. I'm not yet very familiar with the codebase, an alternative to expose a Symbol that can be used in `std` and `cli/web` will work too.

What do you think?